README.md

```diff
@@ -1,244 +1,143 @@
+# `janus-core`
+
+![logo][logo]
+
+[![PyPI version][pypi-badge]][pypi-link]
+[![Python versions][python-badge]][python-link]
 [![Build Status][ci-badge]][ci-link]
 [![Coverage Status][cov-badge]][cov-link]
 [![Docs status][docs-badge]][docs-link]
-[![PyPI version][pypi-badge]][pypi-link]
 [![License][license-badge]][license-link]
 [![DOI][doi-badge]][doi-link]
 
-# janus-core
-
-![logo][logo]
-
 Tools for machine learnt interatomic potentials
 
-## Features in development
+## Getting started
 
```

Comment:
I will add link to the getting started which is in the docs... and of course similar stuff about development, and usage.

I will also add a section something like, janus can be used command line or via python scripting or jupyter notebooks and add link to relevant parts in docs...